descriptors/ModuleDescriptor-template.json

```diff
@@ -487,8 +419,7 @@
         "finance-storage.transactions.collection.get",
         "finance-storage.transactions.item.post",
         "finance-storage.transactions.item.get",
-        "finance-storage.transactions.item.put",
-        "finance-storage.transactions.item.delete"
```

Comment:
Is `finance-storage.transactions.item.delete` permission is not added to `finance-storage.transactions.all` because of following note in [wiki](https://wiki.folio.org/display/FOLIJET/Finance+Data+Model#FinanceDataModel-StorageModule)?
> for internal/emergency admin use only.

Reply:
Now I see a note in PR description :)